sections/email-signup-banner.liquid

```diff
@@ -8,6 +8,27 @@
   #Banner-{{ section.id }}::after {
     opacity: {{ section.settings.image_overlay_opacity | divided_by: 100.0 }};
   }
+
+  #Banner-{{ section.id }} .banner__box {
+    text-align: {{ section.settings.mobile_text_alignment }};
+  }
+
+  @media screen and (min-width: 750px) {
+    #Banner-{{ section.id }} .banner__box {
+      text-align: {{ section.settings.text_alignment }}
+    }
+  }
+
+  #Banner-{{ section.id }} .newsletter-form {
+    display: inline-block;
+  }
+
+  @media screen and (min-width: 750px) {
+    #Banner-{{ section.id }} .banner__content {
+      align-items: {{ section.settings.desktop_text_box_position | split: " " | first }};
+      justify-content: {{ section.settings.desktop_text_box_position | split: " " | last }};
+    }
+  }
```

Comment:
Isn't it needed in case you have the same section twice ? Not sure it's likely to want this email banner section twice on the password page but 🤷 
If we assume that then maybe it could be moved to the CSS file for that specific section 🤔 
It was also to follow a suggestion from Kai on the Image banner PR mentioned in the description.

Reply:
Yes, the reason for this is if we have the same section more than one time, so we can get the respective setting.  

Not likely the merchant would add this section twice, but because theyre able to I think we should account for it

Reply:
I guess the combination of classes would work as well anyway in the CSS. So the main appeal of this was to reduce the amount of CSS lines in accounting for each class (`text-wrapper--top-right`, `text-wrapper--top-left`, `text-wrapper--top-center`, etc).

Reply:
I think this makes sense 🤔 but I do wonder for the other settings since there are so many of them.

If we decide to go that route I believe it would be easier to make that update once I merge this PR to my banner PR